lib/VeresOne.js

```diff
@@ -289,22 +289,21 @@ class VeresOne {
    *
    * @returns {Promise<object>}
    */
-  attachInvocationProof({capability, capabilityAction, operation, key}) {
+  async attachInvocationProof({capability, capabilityAction, operation, key}) {
     const {CapabilityInvocation} = require('ocapld');
     const {suites: {Ed25519Signature2018}} = jsigs;
     return jsigs.sign(operation, {
       documentLoader,
-      suite: new Ed25519Signature2018({
-        key, compactProof: false
-      }),
+      compactProof: false,
```

Comment:
I am reading this as `compactProof` is now an option for `jsigs.sign()` NOT apart of the constructor for `Ed25519Signature2018`. Is that correct?

Reply:
@gannan08 - correct, it's a param for `sign()` (its previous location here was probably just bug/typo)